src/convertToObject.js

```diff
@@ -6,7 +6,57 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString
+    .split(';')
+    .map((str) => str.trim())
+    .filter(Boolean);
+
+  const convertedToObject = {};
+
+  splittedString.forEach((str) => {
+    const styleArr = str.split(':');
+    const [key, value] = styleArr;
+
+    convertedToObject[key.trim()] = value.trim();
```

Comment:
Ensure that both `key` and `value` are defined before attempting to trim and assign them to the `convertedToObject`. If `styleArr` does not contain exactly two elements, this could lead to undefined behavior.